src/main/java/com/sforce/ws/codegen/Generator.java

```diff
@@ -227,12 +226,7 @@ protected TypeMetadataConstructor newTypeMetadataConstructor(Types types, Schema
 
     protected void generateClassFromSimpleType(Schema schema, SimpleType simpleType, File dir) throws IOException {
         SimpleClassMetadata gen = new SimpleClassMetadata(schema, simpleType, typeMapper);
-        ST template;
-        if (gen.getClassName().equals("OrganizationValue")) {
-            template = templates.getInstanceOf(SIMPLE_TYPE_ORGANIZATION_VALUE);
-        } else {
```

Comment:
Screenshot after revert build is fine :
<img width="1395" alt="image" src="https://github.com/user-attachments/assets/8990dfe2-facb-461d-b8a6-bd49413ebe84">